src/phase0/state_transition/block_processing.rs

```diff
@@ -198,16 +202,36 @@ fn process_randao<
         MAX_VALIDATORS_PER_COMMITTEE,
         PENDING_ATTESTATIONS_BOUND,
     >,
-    _body: &BeaconBlockBody<
+    body: &BeaconBlockBody<
         MAX_PROPOSER_SLASHINGS,
         MAX_VALIDATORS_PER_COMMITTEE,
         MAX_ATTESTER_SLASHINGS,
         MAX_ATTESTATIONS,
         MAX_DEPOSITS,
         MAX_VOLUNTARY_EXITS,
     >,
-    _context: &Context,
+    context: &Context,
 ) -> Result<(), Error> {
+    let mut epoch = get_current_epoch(state, context);
+
+    let proposer_index = get_beacon_proposer_index(state, context)?;
+    let proposer = &state.validators[proposer_index];
+
+    let domain = get_domain(state, DomainType::Randao, Some(epoch), context)?;
+    let signing_root = compute_signing_root(&mut epoch, domain)?;
+
+    if !body
+        .randao_reveal
+        .verify(&proposer.pubkey, signing_root.as_ref())
```

Comment:
nitpick, but i think `as_bytes` is a bit more semantically aligned w/ what `verify` wants, rather than simply a reference to the root

Reply:
Fixed: dc8ad40